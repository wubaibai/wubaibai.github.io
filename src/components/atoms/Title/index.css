.title {
	font-family: 'Montserrat', sans-serif;
	font-size: 48px;
	font-weight: 600;
	line-height: 1.2;
}
