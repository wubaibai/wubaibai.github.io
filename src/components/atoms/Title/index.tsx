import React from 'react';
import classNames from 'classnames';

import style from './index.css';

interface TitleProperty {
	color?: string;
}

const Title: React.FC<TitleProperty> = ({ children, color }) => (
	<div className={classNames(style.title, color)}>{children}</div>
);

export default Title;
