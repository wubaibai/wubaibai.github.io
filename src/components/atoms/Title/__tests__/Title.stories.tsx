import React from 'react';
import { storiesOf } from '@storybook/react';

import Title from 'components/atoms/Title';

const stories = storiesOf('atoms/Title', module);

stories.add('__interactive', () => <Title>About</Title>, { jest: 'Title' });
