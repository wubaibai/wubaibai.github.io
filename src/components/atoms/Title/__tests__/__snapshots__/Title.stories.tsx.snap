// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="title"
  >
    About
  </div>
</DocumentFragment>
`;
